utils/docker/Dockerfile.centos.7

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2018-2019, Intel Corporation
+#
+# 'recipe' for Docker to build an image of centOS-based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+FROM centos:7
+MAINTAINER Johann Lombardi <johann.lombardi@intel.com>
+
+# Build arguments can be set via -build-arg
+# use same UID as host and default value of 1000 if not specified
+ARG UID=1000
+
+# Update distribution
+#Nothing to do for CentOS
```

Comment:
CaRT Dockerfile does a `yum clean metadata` here.  Not sure I agree that it should be done, but just that the two projects' Dockerfiles should be equivalent at this point (and beyond) for efficient cache use.

Reply:
Fixed in next update